packages/sources/web-components/src/components/vtmn-badge/vtmn-badge.tsx

```diff
@@ -0,0 +1,40 @@
+import { Component, Prop, h, ComponentInterface } from '@stencil/core';
+
+@Component({
+  tag: 'vtmn-badge',
+  styleUrl: 'vtmn-badge.pcss',
+})
+export class VtmnBadge implements ComponentInterface {
+  /**
+   * The variant of the badge.
+   * @default null
+   */
+  @Prop() variant: 'default' | 'brand' | 'reversed' | 'accent';
+
+  /**
+   * The size of the badge.
+   * @default null
+   */
+  @Prop() size: 'small' | 'medium';
+
+  /**
+   * The target of the badge
+   * @default null
+   */
+  @Prop() value: string;
+
+  render() {
+    return (
+      <div
+        class={[
+          'vtmn-badge',
+          this.size && `vtmn-badge_size--${this.size}`,
+          this.variant && `vtmn-badge_variant--${this.variant}`,
+        ]
+          .filter(Boolean)
+          .join(' ')}>
+        {this.size != 'small' && this.value ? this.value : null}
```

Comment:
@lauthieb I didn't put logic condition for the value. I suppose the developpers will know the DO/Don't so that they won't put text in the badge. Tell me whether I need to condition this or not :)